src/test/java/redis/clients/jedis/commands/commandobjects/CommandObjectsStreamCommandsTest.java

```diff
@@ -885,10 +947,10 @@ public void testXInfoConsumersWithActiveConsumers() {
     Map<String, String> messageBody1 = Collections.singletonMap("field1", "value1");
     Map<String, String> messageBody2 = Collections.singletonMap("field2", "value2");
 
-    exec(commandObjects.xadd(streamKey, StreamEntryID.NEW_ENTRY, messageBody1));
-    exec(commandObjects.xadd(streamKey, StreamEntryID.NEW_ENTRY, messageBody2));
+    exec(commandObjects.xadd(streamKey, (StreamEntryID) null, messageBody1));
+    exec(commandObjects.xadd(streamKey, (StreamEntryID) null, messageBody2));
 
-    exec(commandObjects.xgroupCreate(streamKey, group, new StreamEntryID(), true));
+    exec(commandObjects.xgroupCreate(streamKey, group, null, true));
```

Comment:
Seems unnecessary.

Reply:
There are null checks inside the methods, so this brings more branch coverage.

Reply:
You can do those in `xadd` and `xgroupCreate` specific tests if you want.

Reply:
I moved out the part for `xadd`. `xgroupCreate` I leave for a later PR.